docs/readthedocs/config.md

```diff
@@ -204,20 +204,28 @@ Tba:
 
 ## Grid configuration 
 
+The reference system contains a list of voltage levels. Each element includes the nominal apparent power, the nominal 
+voltage and the separate configuration of each voltage level. The voltage level configuration is composed of the identifier 
+and the nominal voltage.
+
+The configuration of a reference system is optional. If no configuration is provided by the user, the default
+[reference system](models/reference_system) that includes all common german voltage levels is used. For those users 
```

Comment:
Not super important but I think we already use (sphinx-hoverxref)[https://sphinx-hoverxref.readthedocs.io/en/latest/] within the documentation. So we could provide a hovered preview here as well? (This might be moved into a new Issue for some future HiWi)

Reply:
Moved into future issue